packages/expo-sqlite/package.json

```diff
@@ -51,7 +51,8 @@
     "preset": "expo-module-scripts"
   },
   "dependencies": {
-    "@expo/websql": "^1.0.1"
+    "@expo/websql": "^1.0.1",
+    "expo-file-system": "^17.0.1"
```

Comment:
having expo-file-system dependency may bring multiple expo-file-system versions in node_modules if people don't setup version correctly. i would recommend not introducing the other expo modules dependencies inside an expo module.

i looked into your pr and it turns out you were using expo-file-system only to get the document directory path. how about serving the feature from expo-sqlite directly? you can use [Constants](https://docs.expo.dev/modules/module-api/#constants)